### DISCUSSION TITLE
Delay between switching to a tab and loading it

### DISCUSSION THREAD

**Discussion by ChaoticGoodboi** [minbrowser/min#2523] [Ideas]
If a user has an old session with a lot of tabs open, the tabs they need are often in different places, and to get there mouseless one needs to switch between several tabs, often just holding the Go to Left Tab combination and "activating" unwanted tabs for the session and increasing the load on memory, AFAIK. It would be nice if  switching between tabs didn't mean loading them at once like that.

**Comment by ioc2e3** [comment]
> ... . It would be nice if switching between tabs didn't mean loading them at once like that.

Same opinion ...

.
